content/release-notes/1.47.0.md

```diff
@@ -0,0 +1,16 @@
+---
+date: 2021-07-02T19:21:14-05:00
+linktitle: "1.47.0"
+title: "1.47.0"
+kubernetes: "1.18, 1.19, and 1.20"
+---
+
+{{<features>}}
+* Added the ability to opt-in to removing the MinIO object store dependency for KOTS for [kURL embedded](https://kurl.sh/docs/add-ons/kots#advanced-install-options) and [existing cluster](https://kots.io/kotsadm/updating/updating-admin-console/#online-installations) upgrades. If this opt-in is selected as part of the upgrade, KOTS will migrate all application archives and support bundles to a blob/filesystem that is part of the new KOTS Admin Console statefulset. This option is also available for new KOTS installs on [kURL embedded](https://kurl.sh/docs/add-ons/kots#advanced-install-options) and [existing clusters](https://kots.io/kotsadm/installing/online-install/#kots-install). When specified on new installs, MinIO will not be used by KOTS for application archive or support bundle storage.
```

Comment:
The last sentence is a little confusing to me. If specified on new installs, will MinIO still be installed and not used? Or not installed? It reads like we will still install but simply not use it.